### ISSUE TITLE
Symbols not showing in eps format

### ISSUE THREAD

**Issue by meylone** [matplotlib/matplotlib#11372]
### Symbols are missing from eps format

**When I save a figure in eps format, the symbols that have no facecolor are not showing. This happened after I updated to the latest version.**

**Code for reproduction**

```python
import matplotlib.pyplot as plt
plt.plot([1,2,3],[2,1,3],'ko',markerfacecolor='None')
plt.plot([1,2,3],[2,1,3],'k')
plt.savefig('not_working.eps')
plt.savefig('working.eps')
```

I would expect empty circles (as will show if you save it as png), but they are missing when saved as eps.

Unfortunately I don't remember the last version I was using and was working before updating.

  * Operating system: Windows 7
  * Matplotlib version: 2.2.2
  * Matplotlib backend: Qt5Agg
  * Python version: 3.6.5
  * Spyder version: 3.2.8

I did a batch update and didn't notice this only a week after.

**Comment by afvincent** [comment]
@meylone Thank you for the ticket :).

I can reproduce the issue with both 'Qt5Agg' and 'TkAgg' on Fedora 28 and Python 3.6 from conda, on Matplotlib 2.2.2 from conda, **but not on master**. So it looks like it has been fixed. Let's see if I can find when it was done.

For the record, I used the following snippet
```python
import matplotlib as mpl
mpl.use("Qt5Agg")
#mpl.use("TkAgg")
import matplotlib.pyplot as plt

fig, ax = plt.subplots()

ax.plot([2, 1, 3], 'ko', mfc='None', label="mfc='none'")
ax.plot([1, 3, 2], 'ko', label="default mfc")
ax.legend()

fig.savefig('example.eps')
fig.savefig('example.png')
```

**Comment by afvincent** [comment]
Found it! Seems similar to #10921, which has been fixed by @jklymak in #10935. Closing the current issue but anyone feels free to reopen if you think it should have been closed.

**Comment by tacaswell** [comment]
Looks like it has already been backported to 2.2.x and will be in the 2.2.3 release.
